readme.md

```diff
@@ -2,43 +2,40 @@
 
 > [Kap](https://github.com/wulkano/kap) plugin - Upload recordings to [Dropbox](https://dropbox.com)
 
-
 ## Install
 
-In the `Kap` menu, go to `Preferences…`, select the `Plugins` pane, find this plugin, and click `Install`.
-
+In the `Kap` menu, go to `Preferences…`, select the `Plugins` pane, find this plugin, and toggle it.
 
 ## Usage
 
 In the editor, after recording, select a format, and then `Upload to Dropbox`.
 
 ## Options
 
-In the `plugins` pane, click `Open plugins folder` and edit the `kap-dropbox.json` file to change options
+In the “Plugins” pane, click the pencil icon to edit the plugin options.
 
-Available options:
+Alternatively, you can manually edit the options in your text editor; Click `Open plugins folder` and edit the `kap-dropbox.json` file to change options.
 
 #### path
-Type: `String`
 
-When provided, uploads will be placed in that directory. For example, to upload recordings under a directory called Kaptures, set `path` to `/Kaptures`. If the directory doesn't exist it will be created.
+Type: `string`
+
+When provided, uploads will be placed in that directory. For example, to upload recordings under a directory called `Kaptures`, set `path` to `/Kaptures`. If the directory doesn't exist it will be created.
 
 #### autorename
-Type: `Boolean`
+
+Type: `boolean`\
 Default: `true`
 
-When set to true, if there is a conflict (the file already exists) the new upload will automatically be renamed in order to keep both
+If there is a conflict (the file already exists), the new upload will automatically be renamed in order to keep both.
 
 #### mute
-Type: `Boolean`
+
+Type: `boolean`\
 Default: `false`
 
-When set to true, the dropbox app(s) won't show a notification for this upload
+The Dropbox app won't show a notification for this upload.
 
 Used to avoid this:
 
 ![double](media/double.png)
-
-## License
-
-MIT © [George Karagkiaouris](https://gkaragkiaouris.com)
```

Comment:
Removed it as it moot now that GitHub shows the license in the UI itself. I'm removing it from all my readme's too. Let me know if you still want to keep it though.